test/test_with_xpress.jl

```diff
@@ -32,6 +32,170 @@ using Random
+        if i == 3
+            inputs = JSON.parsefile("./scenarios/re_emissions_with_thermal.json")
+        else
+            inputs = JSON.parsefile("./scenarios/re_emissions_elec_only.json")
+        end
+        inputs["Site"]["include_exported_renewable_electricity_in_total"] = include_exported_RE_in_total[i]
+        inputs["Site"]["include_exported_elec_emissions_in_total"] = include_exported_ER_in_total[i]
+        inputs["Site"]["renewable_electricity_min_pct"] = if isnothing(RE_target[i]) 0.0 else RE_target[i] end
+        inputs["Site"]["renewable_electricity_max_pct"] = RE_target[i]
+        inputs["Site"]["CO2_emissions_reduction_min_pct"] = ER_target[i]
+        inputs["Site"]["CO2_emissions_reduction_max_pct"] = ER_target[i]
+        if with_outage[i]
+            outage_start_hour = 4032
+            outage_duration = 2000 #hrs
+            inputs["ElectricUtility"]["outage_start_time_step"] = outage_start_hour + 1
+            inputs["ElectricUtility"]["outage_end_time_step"] = outage_start_hour + 1 + outage_duration
+            inputs["Generator"]["max_kw"] = 20
+            inputs["Generator"]["existing_kw"] = 2
+            inputs["Generator"]["fuel_avail_gal"] = 1000 
+        end
+
+        m1 = Model(optimizer_with_attributes(Xpress.Optimizer, "OUTPUTLOG" => 0, "MAXIIS" => -1))
+        m2 = Model(optimizer_with_attributes(Xpress.Optimizer, "OUTPUTLOG" => 0, "MAXIIS" => -1))
+        results = run_reopt([m1, m2], inputs)
+
+        if !isnothing(ER_target[i])
+            ER_pct_out = results["Site"]["lifecycle_emissions_reduction_CO2_pct"]
+            @test ER_target[i] ≈ ER_pct_out atol=1e-3
+            lifecycle_emissions_tCO2_out = results["Site"]["lifecycle_emissions_tCO2"]
+            lifecycle_emissions_bau_tCO2_out = results["Site"]["lifecycle_emissions_tCO2_bau"]
+            ER_pct_calced_out = (lifecycle_emissions_bau_tCO2_out-lifecycle_emissions_tCO2_out)/lifecycle_emissions_bau_tCO2_out
+            ER_pct_diff = abs(ER_pct_calced_out-ER_pct_out)
+            @test ER_pct_diff ≈ 0.0 atol=1e-2
+        end
+        year_one_emissions_tCO2_out = results["Site"]["year_one_emissions_tCO2"]
+        yr1_fuel_emissions_tCO2_out = results["Site"]["year_one_emissions_from_fuelburn_tCO2"]
+        yr1_grid_emissions_tCO2_out = results["ElectricUtility"]["year_one_emissions_tCO2"]
+        yr1_total_emissions_calced_tCO2 = yr1_fuel_emissions_tCO2_out + yr1_grid_emissions_tCO2_out 
+        @test year_one_emissions_tCO2_out ≈ yr1_total_emissions_calced_tCO2 atol=1e-1
+        if !isnothing(results["Financial"]["breakeven_cost_of_emissions_reduction_per_tCO2"])
+            @test results["Financial"]["breakeven_cost_of_emissions_reduction_per_tCO2"] >= 0.0
+        end
+        
+        #commented out values are using levelization factors matching API
+        if i == 1
+            @test results["PV"]["size_kw"] ≈ 45.9 atol=1e-1#46.3 atol=1e-1
+            @test results["Wind"]["size_kw"] ≈ 10.0
```

Comment:
can we remove Wind from these tests? When developing we were often hitting the WindToolKit API request limit due to many Wind tests (which we since reduced to one Wind test).